tests/existing-resources/main.tf

```diff
@@ -0,0 +1,43 @@
+module "resource_group" {
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.6"
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+#############################################################################
+# Provision VPC
+#############################################################################
+
+module "vpc" {
+  source            = "terraform-ibm-modules/landing-zone-vpc/ibm"
+  version           = "7.19.1"
+  resource_group_id = module.resource_group.resource_group_id
+  region            = var.region
+  name              = "vpc"
+  prefix            = var.prefix
+  tags              = var.resource_tags
+  subnets = {
+    zone-1 = [
+      {
+        name           = "subnet-a"
+        cidr           = "10.10.10.0/24"
+        public_gateway = true
+        acl_name       = "vpc-acl"
+      }
+    ]
+  }
+}
+
+data "ibm_is_image" "image" {
+  name = var.image_name
+}
+
+module "secrets_manager" {
```

Comment:
Secrets Manager is optional for the DA so I would not provision this as part of DA validation. For other tests we should re-use our permanent secrets manager instance instead of creating a new one to help reduce the quota limit we regularly hit